lib/src/runner/remote_listener.dart

```diff
@@ -105,7 +124,7 @@ class RemoteListener {
   static void _sendError(StreamChannel channel, error, StackTrace stackTrace) {
     channel.sink.add({
       "type": "error",
-      "error": RemoteException.serialize(error, stackTrace)
+      "error": RemoteException.serialize(error, testChain(stackTrace))
```

Comment:
This should probably choose whether to produce a verbose trace based on the metadata once it's available, and default to verbose before then.